appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -786,7 +800,7 @@ public void onSuccess(String json) {
               }
             });
             editorManager = new EditorManager();
-
+            bindProjectListbox = ProjectListBox.getProjectListBox();
```

Comment:
Is there any reason in particular to do this here rather than just assign it at the field initializer on line 218?

Reply:
> Is there any reason in particular to do this here rather than just assign it at the field initializer on line 218?

It should be. This can go.